link/src/main/java/com/stripe/android/link/ui/LinkIcon.kt

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android.link.ui
+
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.stripe.android.uicore.R
+
+@Composable
+internal fun LinkIcon(
+    modifier: Modifier = Modifier,
+    tint: Color = Color.Unspecified
```

Comment:
Why is this a variable? Should this value always be the link logo color?

Reply:
Since we're using an icon and not an image, any tint other than `Color.Unspecified` changes the color of the logo. `LinkButton` needs the tint to be `Color.Unspecified`